tools/src/main/scala/com/nvidia/spark/rapids/tool/profiling/GenerateDot.scala

```diff
@@ -292,6 +284,39 @@ object SparkPlanGraph {
             exchanges, accumIdToStageId, stageIdToStageMetrics))
     }
   }
+
+  def createDotLabel(
+    outputDir: String, 
+    appId: String, 
+    sqlId: String,
+    physicalPlan: String,
+    maxLength: Int = 16384 
+  ): String = {
+    val sqlPlanFile = s"file://$outputDir/planDescriptions-$appId"
```

Comment:
outputDir isn't necessarily file://, could be in hdfs for instance, I guess maybe you are thinking they download it anyway so when open it should be?  But if they do that outputdir could be different

Reply:
yes forgot about non-OS-mounted locations.